### PR TITLE
zoom bombing document

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Please review PR AND the document linked.

### CODE REVIEW

File: docs/how-to-handle-public-calendar.md [hunk 1]
```diff
@@ -37,4 +37,13 @@ meetings can join the group.
...
-You can request to be added to this Google Group by creating an issue in this repository.
+If your identity is not recognizable from the e-mail you are using to request joining the group, please
+request to be added to this Google Group by creating an issue in this repository.
+
+Important: all members of this Google Group has access to [zoom meetings host keys](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#).
+Please keep the membership of this group up to date and accurate.
+
+## Zoom bombing
+
+All members of opentelemetry-calendar-contributors@googlegroups.com has access to [the document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
```
Comment (bhs) [inline docs/how-to-handle-public-calendar.md:48]:
```suggestion
All members of opentelemetry-calendar-contributors@googlegroups.com has access to [this document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
```

**Review by alolita — APPROVED** [review]

**Review by alolita — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
@mtwo please communicate this calendar with maintainers on Monday as part of adding passwords to zoom rooms

File: docs/how-to-handle-public-calendar.md [hunk 2]
```diff
@@ -37,4 +37,13 @@ meetings can join the group.
...
 All SIG maintainers have permission to edit the Public OpenTelemetry calendar.
 To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com.
-You can request to be added to this Google Group by creating an issue in this repository.
+If your identity is not recognizable from the e-mail you are using to request joining the group, please
+request to be added to this Google Group by creating an issue in this repository.
+
+Important: all members of this Google Group has access to [zoom meetings host keys](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#).
+Please keep the membership of this group up to date and accurate.
+
+## Zoom bombing
```
Comment (arminru) [inline docs/how-to-handle-public-calendar.md:46]:
```suggestion
## Zoom bombing prevention
```

File: docs/how-to-handle-public-calendar.md [hunk 3]
```diff
@@ -37,4 +37,13 @@ meetings can join the group.
...
+
+All members of opentelemetry-calendar-contributors@googlegroups.com has access to [this document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
```
Comment (arminru) [inline docs/how-to-handle-public-calendar.md:48]:
```suggestion
All members of opentelemetry-calendar-contributors@googlegroups.com have access to [this document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
```

File: docs/how-to-handle-public-calendar.md [hunk 4]
```diff
@@ -37,4 +37,13 @@ meetings can join the group.
 
 All SIG maintainers have permission to edit the Public OpenTelemetry calendar.
 To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com.
-You can request to be added to this Google Group by creating an issue in this repository.
+If your identity is not recognizable from the e-mail you are using to request joining the group, please
+request to be added to this Google Group by creating an issue in this repository.
+
+Important: all members of this Google Group has access to [zoom meetings host keys](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#).
```
Comment (arminru) [inline docs/how-to-handle-public-calendar.md:43]:
```suggestion
```
repetition of the below

**Review by arminru — APPROVED** [review]

File: docs/how-to-handle-public-calendar.md [hunk 5]
```diff
@@ -37,4 +37,12 @@ meetings can join the group.
 
 All SIG maintainers have permission to edit the Public OpenTelemetry calendar.
 To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com.
```
Comment (arminru) [inline docs/how-to-handle-public-calendar.md:39]:
```suggestion
To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com by submitting a request [here](https://groups.google.com/g/opentelemetry-calendar-contributors).
```

File: docs/how-to-handle-public-calendar.md [hunk 6]
```diff
@@ -37,4 +37,12 @@ meetings can join the group.
...
-You can request to be added to this Google Group by creating an issue in this repository.
+If your identity is not recognizable from the e-mail you are using to request joining the group, please
+request to be added to this Google Group by creating an issue in this repository.
+
+Please keep the membership of this group up to date and accurate.
+
+## Zoom bombing prevention
+
+All members of opentelemetry-calendar-contributors@googlegroups.com have access to [this document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
+explaining how to deal with inappropriate behavior in Zoom.
```
Comment (arminru) [inline docs/how-to-handle-public-calendar.md:48]:
```suggestion
listing the host keys for our meetings and explaining how to deal with inappropriate behavior in Zoom.
```

**Review by arminru — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
@arminru your suggestions are great as always! Thank you for review

**Review by dyladan — APPROVED** [review]

File: docs/how-to-handle-public-calendar.md [hunk 7]
```diff
@@ -36,5 +36,13 @@ as a participant. Anybody who wants to recieve up to date invites for all OpenTe
...
-To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com.
-You can request to be added to this Google Group by creating an issue in this repository.
+To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com by submitting a request [here](https://groups.google.com/g/opentelemetry-calendar-contributors).
+If your identity is not recognizable from the e-mail you are using to request joining the group, please
+request to be added to this Google Group by creating an issue in this repository.
+
+Please keep the membership of this group up to date and accurate.
+
+## Zoom bombing prevention
+
```
Comment (mtwo) [inline docs/how-to-handle-public-calendar.md:46]:
```suggestion
All meetings now require a passcode to join: _77777_. This passcode is embedded into the calendar meeting invites and meeting notes documents.
```

**Review by mtwo — COMMENTED** [review]
Added a line for the passcode

**Review by mtwo — APPROVED** [review]

**Pull request merged**